boards/FSM/Src/Io/Io_SteeringAngleSensor.c

```diff
@@ -5,7 +5,7 @@ float Io_SteeringAngleSensor_GetAngleDegree(void)
 {
     // Steering angle = (ADC Voltage - Voltage Offset) * Degree Per Volt
 
-    const float STEERING_ANGLE_VOLTAGE_OFFSET = 1.9f;
+    const float STEERING_ANGLE_VOLTAGE_OFFSET = 1.85f;
```

Comment:
this offset can be done with hardware, try to look into `Parameter Settings` under for this ADC in Cube

Reply:
to me it seems clearer to do offset via code, is there a reason to do in cube?

Reply:
Less work done by software, more work done by peripheral so it's better to use thr HW offset

Reply:
lets do this in a later PR though